.github/workflows/cool.yml

```
@@ -1,182 +1,213 @@
 name: COOL Deploy
 on:
+  pull_request:
+    branches: [develop]
   push:
     branches: [develop]
   repository_dispatch:
     types: [deploy, cool_staging, cool_production]
 
 env:
-  AWS_ACCESS_KEY_ID: ${{ secrets.COOL_AWS_ACCESS_KEY_ID }}
-  AWS_SECRET_ACCESS_KEY: ${{ secrets.COOL_AWS_SECRET_ACCESS_KEY }}
-  PRODUCTION_ACCOUNT_ID: "779871898657"
-  STAGING_ACCOUNT_ID: "719488448316"
+  # AWS
   REGION: us-east-1
-  AWS_REGION: us-east-1
-  AWS_DEFAULT_REGION: us-east-1
+
+  # COOL Account IDs
+  PRODUCTION_ACCOUNT_ID: ${{ secrets.COOL_AWS_PRODUCTION_ACCOUNT_ID }}
+  STAGING_ACCOUNT_ID: ${{ secrets.COOL_AWS_STAGING_ACCOUNT_ID }}
+
+  # Shared Services Account IDs
+  SHARED_SERVICES_PRODUCTION_ACCOUNT_ID: ${{ secrets.COOL_SHARED_SERVICES_PRODUCTION_ACCOUNT_ID }}
+  SHARED_SERVICES_STAGING_ACCOUNT_ID: ${{ secrets.COOL_SHARED_SERVICES_STAGING_ACCOUNT_ID }}
+
+  # Github repositories
+  API_IMAGE_REPO: domain-manager-api
+  UI_IMAGE_REPO: domain-manager-ui
+
+  # Booleans
+  IS_STAGING: ${{ github.event_name == 'push' || github.event.action == 'cool_staging' || github.event.action == 'deploy' }}
+  IS_PRODUCTION: ${{ github.event.action == 'cool_production' }}
+  IS_PULL_REQUEST: ${{ github.event_name == 'pull_request' }}
+  BUILD_UI: github.event.client_payload.trigger == env.UI_IMAGE_REPO || github.event.action == 'cool_production'
+  BUILD_API: github.event.client_payload.trigger == env.API_IMAGE_REPO || github.event.action == 'cool_production'
 
 jobs:
-  setenv:
+  deploy:
     runs-on: ubuntu-latest
-    outputs:
-      ENVIRONMENT: ${{ steps.env.outputs.ENVIRONMENT }}
-      ASSUME_ROLE: ${{ steps.env.outputs.ASSUME_ROLE }}
-      ECR_URI: ${{ steps.env.outputs.ECR_URI }}
     steps:
+      # Install Terraform
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.12.30
```

Might as well bump up to 0.12.31...
```suggestion
          terraform_version: 0.12.31
```

---

@dav3r Fixed in https://github.com/cisagov/domain-manager-cicd/pull/11/commits/f837a71e217904a2726fec1ac3a4581a493180ac